lib/proxy/settings.rb

```diff
@@ -10,6 +10,7 @@ module Proxy::Settings
   def self.initialize_global_settings(settings_path = nil, argv = ARGV)
     global = ::Proxy::Settings::Global.new(YAML.load(File.read(settings_path || SETTINGS_PATH)))
     global.apply_argv(argv)
+    global.apply_env(ENV)
```

Comment:
Would it make sense to make this a parameter as well, just like argv so it's easier to test?

Reply:
It already is a parameter, for exactly that reason. ;-)